src/screens/pause.py

```diff
@@ -17,7 +16,7 @@ def button_action(self, text):
         if text == 'Options':
             self.switch_screen(GameState.SETTINGS)
         if text == "Save and Resume":
-            self.switch_screen(GameState.LEVEL)
+            self.switch_screen(GameState.SAVE_AND_RESUME)
```

Comment:
i also don't understood the save and resume button because when you click on back button in setting menu this already save settings

Reply:
It's not the same thing. This is saving the game itself, not the keybinds

Reply:
okay nice